apps/upload-log/upload-log-cf.yml

```
@@ -7,11 +7,16 @@ Parameters:
 
 Outputs:
 
-  LambdaArn:
-    Value: !GetAtt Lambda.Arn
+  UploadLogActivity:
+    Value: !Ref UploadLogActivity
 
 Resources:
 
+  UploadLogActivity:
+    Type: AWS::StepFunctions::Activity
+    Properties:
+      Name: !Sub "${AWS::StackName}-upload-log-activity"
```

The lambda IAM role will need additional permissions to interact with the activity

---

I expect `upload-log` will be in the stack name, so explicitly inluding it might be redundant
```suggestion
  UploadLogActivity:
    Type: AWS::StepFunctions::Activity
    Properties:
      Name: !Sub "${AWS::StackName}-activity"
```